ncrystal_python/src/NCrystal/ncmat2endf.py

```diff
@@ -52,15 +52,10 @@ def import_endfparserpy():
         #from endf_parserpy.interpreter.fortran_utils import read_fort_floats
         #from endf_parserpy.interpreter.fortran_utils import write_fort_floats
     except ImportError:
-        raise SystemExit('Could not import endf_parserpy. Check the package '+
-                         'was correctly installed, with a version equal or '+
-                         'higher than 0.11.0.\n'+
+        raise SystemExit('Could not import endf_parserpy. Check the package '
+                         'was correctly installed, with a version equal or '
+                         'higher than 0.11.0.\n'
```

Comment:
It still says 0.11.0 here, I think we should just mention that people can install with:
1. pip install ncrystal[endf] or ncrystal[all]
2. pip install endf_parserpy
3. conda install -c conda-forge endf_parserpy
And not talk about versions (because in 6 months those versions will be out of sync with what it says in the dependency database).